### DISCUSSION TITLE
Low quality transcript - bug?

### DISCUSSION THREAD

**Discussion by scoutaroom** [deepgram/community#576] [General help]
### Which Deepgram product are you using?

None

### Details

Hey 
We are testing the NOVA 2 medical and today the transcript is very low... is it only us?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by scoutaroom** [reply]
POST /v1/listen?model=nova-2-medical&smart_format=true&diarize=true&callback=https%3A%2F%[2Fzirr.ai](http://2fzirr.ai/)%2Faudioapi%2Fcallback%2F65babd47372193b8abd82e31

We use  Node SDK and we use  deepgram.listen.prerecorded.transcribeFileCallback
